apartmenttherapy/order-service
# Elaine/sendgrid

#### What does this PR do & why?
Sorry, this is a big one. This adds all the functionality to send all the emails.

#### How can this be manually tested?
- Tests should run.
- Should see emails on the following events:
  - New order: seller & buyer
  - Order accepted: buyer
  - Order rejected: buyer
  - Order paid: seller
  - Order completed: buyer
  - Order canceled by seller: buyer
  - Order canceled by buyer: seller

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*


#### Link to story card and other documentation
https://app.asana.com/0/694906336753887/963025134295735

#### GIF


#### Have you remembered to...?
